tests/unit/requestMulti.test.ts

```diff
@@ -129,19 +133,19 @@ describe("nats-request-multi tests", () => {
     });
 
     it("returns any NATS errors when expected > 1 and received === expected", async () => {
-      const natsError = { code: "err", name: "err", message: "test error" };
+      const natsError = { ...natsErrorBase, code: "err", name: "err", message: "test error" };
       try {
-        await testRequestMulti(3, 3, { natsError: natsError });
+        await testRequestMulti(3, 3, ({ natsError }));
```

Comment:
Hm, I'm not sure you need the parentheses here, but if it works it's not a big deal.